modules/ROOT/pages/backup-restore/planning.adoc

```diff
@@ -184,18 +169,44 @@ They must be run directly on the databases that are associated with that Composi
 | {check-mark}
 |===
 
+
+[NOTE]
+====
+The Neo4j Admin commands `backup`, `restore`, `dump`, `load`, `copy`, and `check-consistency` are not supported for use on xref:database-administration/composite-databases/concepts.adoc[Composite databases].
+They must be run directly on the databases that are associated with that Composite database.
+====
+
+
+== Considerations for backing up and restoring databases in a cluster
+
+Backing up a database in a clustered environment is not essentially different from a standalone backup, apart from the fact that you must know which server in a cluster to connect to.
+Use `SHOW DATABASE <database>` to learn which servers are hosting the database you want to back up.
+See xref:clustering/monitoring/show-databases-monitoring.adoc#show-databases-monitoring-listing-single[Listing a single database] for more information.
+
+However, _restoring_ a database in a cluster is different since it is not known in advance how a database is going to be allocated to the servers in a cluster.
+This method relies on the seed already existing on one of the servers.
+The recommended way to restore a database in a cluster is to xref::database-administration/standard-databases/seed-from-uri.adoc[seed from URI].
+
+[IMPORTANT]
+====
+When backing up a clustered database, be aware that the xref:clustering/introduction.adoc#clustering-introduction-operational[cluster topology] metadata is not included in the backup.
+The cluster topology is stored in the `system` database and describes how the copies of a database should be spread across the servers in a cluster.
+
+Because this metadata is excluded from backups, restoring to a new cluster environment requires manual reconfiguration of servers modes and database allocations. 
```

Comment:
This seems a bit negative about the topology not being backed up. I think it's actually a positive. Perhaps something more along the lines of:

> Backing up a database only includes the database contents, and the RBAC associated with the database if you include the metadata.
> This means you specify the desired topology when you restore the backup, allowing you to restore with a different topology than was present when backed up.
> The database will be allocated to servers that are present at restore time to fulfil that topology.

I wouldn't mention that the topology is stored in `system`, that's effectively an implementation detail.